docs/Project.toml

```diff
@@ -40,7 +39,7 @@ Clarabel = "=0.5.1"
 DataFrames = "1"
 DimensionalData = "0.24"
 Distributions = "0.25"
-Documenter = "0.27.9, 0.28"
+Documenter = "1"
```

Comment:
```suggestion
Documenter = "=1.0.0"
```

Reply:
We're accessing some of the Documenter internals (like adding a build step), so it's probably best if we lock the exact version. x-ref: https://documenter.juliadocs.org/stable/lib/semver/#What-is-not-covered-by-SemVer